pkg/quota/image/testutil/util.go

```diff
@@ -0,0 +1,300 @@
+      }
+   ]
+}`
+
+func GetFakeImageStreamImageGetHandler(t *testing.T, namespace string, iss ...imageapi.ImageStream) ktestclient.ReactionFunc {
+	tevList := imageapi.TagEventList{}
+
+	for _, imgName := range []string{
+		BaseImageWith1LayerDigest,
+		BaseImageWith2LayersDigest,
+		ChildImageWith2LayersDigest,
+		ChildImageWith3LayersDigest,
+		MiscImageDigest,
+	} {
+		tevList.Items = append(tevList.Items,
+			imageapi.TagEvent{
+				DockerImageReference: fmt.Sprintf("172.30.12.34:5000/test/is@%s", imgName),
+				Image:                imgName,
+			})
+	}
+
+	sharedIS := imageapi.ImageStream{
+		ObjectMeta: kapi.ObjectMeta{
+			Namespace: "shared",
+			Name:      "is",
+		},
+		Status: imageapi.ImageStreamStatus{
+			Tags: map[string]imageapi.TagEventList{
+				"latest": tevList,
+			},
+		},
+	}
+
+	return func(action ktestclient.Action) (handled bool, ret runtime.Object, err error) {
+		switch a := action.(type) {
+		case ktestclient.GetAction:
+			if a.GetNamespace() != namespace && a.GetNamespace() != "shared" {
+				err := fmt.Errorf("namespace %q not found", a.GetNamespace())
+				t.Error(err.Error())
+				return true, nil, err
+			}
+
+			imageStreams := append([]imageapi.ImageStream{sharedIS}, iss...)
+			for _, is := range imageStreams {
+				if !strings.HasPrefix(a.GetName(), is.Name+"@") {
+					continue
+				}
+				name := strings.TrimPrefix(a.GetName(), is.Name+"@")
+
+				res := &imageapi.ImageStreamImage{
+					ObjectMeta: kapi.ObjectMeta{
+						Namespace: namespace,
+						Name:      a.GetName(),
+					},
+					Image: imageapi.Image{
+						ObjectMeta: kapi.ObjectMeta{
+							Name:        name,
+							Annotations: map[string]string{imageapi.ManagedByOpenShiftAnnotation: "true"},
+						},
+						DockerImageReference: fmt.Sprintf("registry.example.org/%s/%s", namespace, a.GetName()),
+					},
+				}
+
+				switch name {
+				case BaseImageWith1LayerDigest:
+					res.Image.DockerImageManifest = BaseImageWith1Layer
+				case BaseImageWith2LayersDigest:
+					res.Image.DockerImageManifest = BaseImageWith2Layers
+				case ChildImageWith2LayersDigest:
+					res.Image.DockerImageManifest = ChildImageWith2Layers
+				case ChildImageWith3LayersDigest:
+					res.Image.DockerImageManifest = ChildImageWith3Layers
+				case MiscImageDigest:
+					res.Image.DockerImageManifest = MiscImage
+				default:
+					err := fmt.Errorf("image %q not found", name)
+					t.Error(err.Error())
+					return true, nil, err
+				}
+
+				t.Logf("imagestreamimage get handler: returning %q", res.Name)
+				return true, res, nil
+			}
+
+			err := fmt.Errorf("imagestreamimage %q not found", a.GetName())
+			t.Error(err.Error())
+			return true, nil, err
+		}
+		return false, nil, nil
+	}
+}
+
+func FakeImageGetHandler(t *testing.T, namespace string) ktestclient.ReactionFunc {
```

Comment:
Missing `Get` prefix as in the above a function. Actually the prefix can be also removed from the function above. Just let them be consistent.